reference/glossary.rst

```diff
@@ -810,13 +810,24 @@ Glossary
         *Work in Progress*
 
     jitter
-        *Work in Progress*
+        Jitter is the variation in delay or latency between when data packets are sent and when they are received over a network, causing irregular arrival times at the destination. This variation is often caused by network congestion, packet loss, poor hardware performance or differences in the path packets take. 
+
+        Related topic(s): :term:`Chrony`.
 
     journald
-        *Work in Progress*
+        Journald is a logging driver that sends container logs to the systemd journal. 
+
+        See `journald <https://docs.docker.com/engine/logging/drivers/journald/>`_ for more details.
```

Comment:
I'd suggest expanding this bit of text to provide a bit more context for the reader on where the link will take them. Something like:

See the `Docker journald documentation` for details on using journald as a logging driver.

Or something like that. What do you think?

Reply:
This would be helpful for sure